riscv/src/bootloader.rs

```diff
@@ -1,21 +1,30 @@
 use number::FieldElement;
 
-// 32-Bit architecture -> 2^32 bytes of addressable memory
+/// 32-Bit architecture -> 2^32 bytes of addressable memory
 pub const MEMORY_SIZE_LOG: usize = 32;
 
-// Page size is 1KB
+/// Page size is 1KB
 pub const PAGE_SIZE_BYTES_LOG: usize = 10;
 
-// 32-Bit architecture -> 4 bytes per word
+/// 32-Bit architecture -> 4 bytes per word
 pub const BYTES_PER_WORD: usize = 4;
 
+/// The root hash of the memory Merkle tree for an empty memory.
+pub const MEMORY_ZERO_HASH: [u64; 4] = [
+    5416844844920273493,
+    433850106913325065,
+    9201603462779782159,
+    8581776654970304227,
+];
```

Comment:
Would it be possible to compute this at compile-time?

Reply:
I guess instead of hard-coding it, we can do `MerkleTree::new().root_hash()`. I did that in db415d667104f75a714bd35936f589b0803ca30a! Think it's better :)